woodwork/table_accessor.py

```diff
@@ -307,7 +307,7 @@ def physical_types(self):
         """A dictionary containing physical types for each column"""
         if self._schema is None:
             _raise_init_error()
-        return {col_name: _get_valid_dtype(type(self._dataframe[col_name]), self.schema.logical_types[col_name]) for col_name in self._dataframe.columns}
+        return {col_name: _get_valid_dtype(type(self._dataframe[col_name]), self._schema.logical_types[col_name]) for col_name in self._dataframe.columns}
```

Comment:
Here and one place above we were using `self.schema` instead of `self._schema`, which is preferable.